README.md

```diff
@@ -1324,6 +1324,7 @@ Table of Contents
   * [ScreenshotAPI.net](https://screenshotapi.net/) - Screenshot API use one simple API call to generate screenshots of any website. Build to scale and hosted on Google Cloud. Offers 100 free screenshots per month.
   * [screenshotlayer.com](https://screenshotlayer.com/) — Capture highly customizable snapshots of any website. Free 100 snapshots/month
   * [screenshotmachine.com](https://www.screenshotmachine.com/) — Capture 100 snapshots/month, png, gif and jpg, including full-length captures, not only home page
+  * [Statically Screenshot](https://statically.io/docs/using-screenshot/) — A free screenshot API to convert web to image or PDF. Supports desktop and mobile view.
```

Comment:
think we can just update the one entry with additional features

Reply:
no problem, I revert the addition for now